.github/workflows/publish-containers.yml

```
@@ -1,25 +1,39 @@
-    # Nightly image rebuild (1h offset from binary workflow to avoid S3 contention).
-    - cron: '0 4 * * *'
   workflow_dispatch:
     inputs:
       therock_version_override:
         description: 'TheRock ROCm version override (e.g., 7.15.0a20250818 or "latest")'
         required: false
         default: 'latest'
+      artifact_sha:
+        description: 'GitHub SHA of the prebuilt artifact to consume (must match a published build-heretek-rocmfpx artifact).'
+        required: false
+        default: ${{ github.sha }}
```

actionlint (1.7.12)

[error] context "github" is not allowed here. no context is available here. see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)